client/my-sites/media-library/content.jsx

```diff
@@ -218,6 +218,14 @@ class MediaLibraryContent extends React.Component {
 		);
 	}
 
+	renderConnectExternalMedia() {
```

Comment:
This needs to return something if not google photos (e.g `null`)

Reply:
Fixed